### DISCUSSION TITLE
Minus sign

### DISCUSSION THREAD

**Discussion by juup** [jomjol/AI-on-the-edge-device#3169] [General]
Due to a solar system, the values of the current electricity consumption can be negative. According to the documentation, however, only 10 digits (0-9) and N are recognized. Is it also possible to recognize the minus sign?

**Comment by caco3** [comment]
no, this currently is not supported

**Comment by zakkieboy44** [comment]
May I ask if it ever will be supported (The Negative Symbol detect) ? Thanks in advance

**Reply by caco3** [reply]
We are a small team and invest only sporadically time into the project. Thus AFAIK nobody is working on this feature, sry.

**Reply by zakkieboy44** [reply]
Thanks for coming back, understand what you mean and that efforts need to be focused.
